users/tests.py

```diff
@@ -43,7 +43,7 @@ def test_signup_post_success(self, mock_boto_client):
         })
 
         form = {
-            'profile_photo': open('/home/yeonu//사진/Screenshot.png','rb'),
+            'profile_photo': open('/home/yeonu/사진/Screenshot.png','rb'),
```

Comment:
나중에는 file 데이터를 테스트하실때는 로컬에 존재하는 파일을 열어서 테스트 하는것이 아니라, 가짜 file을 만들어서 테스트해주셔야되요! 가짜 파일을 만드는 방법을 찾아보시고, test에 적용해보세요~
```suggestion
            'profile_photo': open('/home/yeonu/사진/Screenshot.png','rb'),
```

Reply:
오 그렇군요!! 한번 해보겠습니당~

Reply:
머지 마감세일이 임박해서 리팩토링 때 도전해보겠습니다..ㅎ